src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -127,8 +127,6 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                             return;
                         }
                     }
-                } else {
-                    return;
```

Comment:
I got this in my pr #61 . I suggest to avoid conflict we leave this out of your pr.